app/services/claims/fee_calculator/calculate.rb

```diff
@@ -83,20 +84,41 @@ def fee_scheme
         @fee_scheme ||= client.fee_schemes(type: scheme_type, case_date: earliest_representation_order_date.to_s(:db))
       end
 
+      def bill_scenario
+        CCR::CaseTypeAdapter::BILL_SCENARIOS[case_type.fee_type_code.to_sym]
+      end
+
       # TODO: consider creating a mapping to fee calculator id's
       # - less "safe" but faster/negates the need to query the API??
       #
       def scenario
-        # TODO: create select/find_by calls to list endpoints in client gem
+        # TODO: create select/find_by calls to retrieve endpoint data by attribute value
```

Comment:
relates to [laa-fee-calculator-client PR](https://github.com/ministryofjustice/laa-fee-calculator-client/pull/24)